client/components/UserDecalPanel.js

```diff
@@ -87,6 +90,37 @@ class UserDetailsPanel extends Component {
     return null
   }
 
+  _renderDecalCode = (decal, index) => (
+    <>
+      <div className="decal-code" key={`code${index}`}>
+        <button
+          key={`toggle${index}`}
+          aria-label="toggle-decal"
+          className="icon decal-toggle"
+          onClick={this._handleToggle}
+          // title={maxNicksReached ? 'You\'ve used all your nicknames' : 'Add new nickname'}
+          type="button">
+          <FontAwesomeIcon icon={this.state.formOpen[index] ? 'eye-slash' : 'eye'} fixedWidth />
+        </button>
+        {decal.attributes.code}
+      </div>
+      <div className="decal-claimed-at" key={`claimedAt${index}`}>{formatAsEliteDate(decal.attributes.claimedAt)}</div>
+    </>
+  )
+
+  _renderDecalCodes = () => {
+    const { decals } = this.props
+    console.log(decals)
+
+    if (decals === []) {
```

Comment:
Never compare to `[]` as it will always return false. using `[]` always creates a new instance of an object, meaning regardless of contents the two will never be equal. Since decals is always an array, it's safe to do a truthy check on `decals.length`.
```suggestion
    if (!decals.length) {
```